apps/server-mcp/global.d.ts

```
@@ -0,0 +1,5 @@
+declare module NodeJS {
```

Biome (2.1.2)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)